src/Commands/HydroponicsDrushCommands.php

```diff
@@ -81,31 +81,35 @@ public function hydroponics(
       });
     }
 
-//    $loop->addTimer(720.0, function ($timer) use ($loop, $timer_ping, &$done) {
-    $loop->addTimer(60.0, function ($timer) use ($loop, $timer_ping, &$done, &$idle) {
-
+    $loop->addPeriodicTimer(60.0, function ($timer) use ($loop, $timer_ping, &$done, &$idle) {
       // Finish all if all queues return 0 elements for at least 3 cycles
       // Check this every 60 s
-
       $all_idle = 1;
       foreach($idle as $queue_idle) {
         if ($queue_idle > 0) {
           $all_idle = 0;
         }
       }
-
       if ($all_idle === 1) {
         $loop->cancelTimer($timer_ping);
           foreach($done as $queue_timer) {
             $loop->cancelTimer($queue_timer);
           }
           \Drupal::state()->set('hydroponics.queurunner_last_pid', 0);
         }
```

Comment:
Great! A tiny little thing missing here. 
- After setting the state it needs to cancel itself. `$loop->cancelTimer($timer)`
- And it needs to cancel security one too, so the line `$loop->addTimer(720.0..` needs to be defined before the 60 second one and passed to the 60 seconds one so, something like `$securitytimer = $loop->addTimer(720.0..`. But... here is the complexity? How do we make this last timer suspend also the 60 seconds one?

Reply:
We have to use $loop->stop(); as here https://github.com/esmero/strawberry_runners/blob/25ee9fefbfc6d43942d6640fc930e2ef904f9676/src/Scripts/mainLoop.php#L348
As 720 probably will disappear in the future so I'd add $loop->stop(); to 60s timer and, at the moment, make 60s cancel 720s too. What do you think?